src/main/resources/org/openstreetmap/atlas/geography/atlas/pbf/atlas-edge.json

```diff
@@ -1,6 +1,9 @@
 {
     "filters": [
-        "access->!no",
+        "access->!no|motor_vehicle->yes|motorcar->yes|vehicle->yes",
```

Comment:
It'd be nice to have a comment in `PbfOneWay`'s referencing this file.

Reply:
Good call!